packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { WORKFLOW_MODEL_UID, STAGE_MODEL_UID } = require('../../constants/workflows');
+
+module.exports = ({ strapi }) => ({
+  async bootstrap() {
```

Comment:
I think this is the best way to define the bootstrap for RW right now.
But I also think the EE folder is not structured to be scalable. We could discuss it with more people, but I think we could find a way to define the bootstrap outside a service.

What I miss in the EE folder is separating features in different folders like:
```
.
├── audit-logs/
│   ├── controllers
│   ├── routes
│   └── ...
├── sso/
│   ├── controllers
│   ├── routes
│   └── ...
└── review-workflows/
    ├── controllers
    ├── routes
    └── ...
```

Reply:
I agree, we could have a more modular folder approach. But I don't know if we want to organize our code like that. Should it be a collegial decision? Like, BE sync?

Reply:
agree, this should be in the BE sync. Let's bring the topic next friday :)